website/docs/configuration/resources.html.md

```diff
@@ -229,6 +229,38 @@ resource "aws_instance" "app" {
 }
 ```
 
+To reference a particular instance of a resource you can use `resource.foo.*.id[#]` where `#` is the index number of the instance.
+
+For example, to create a list of all [AWS subnet](/docs/providers/aws/r/subnet.html) ids vs referencing a specific subnet in the list you can use this syntax:
+
+```hcl
+resource "aws_vpc" "foo" {
+  cidr_block = "198.18.0.0/16"
+}
+
+resource "aws_subnet" "bar" {
+  count      = 2
+  vpc_id     = "${aws_vpc.foo.id}"
+  cidr_block = "${cidrsubnet(aws_vpc.foo.id, 8, count.index)}"
```

Comment:
This was my mistake in my example :confounded: but with fresh eyes I see that this should be `aws_vpc.foo.cidr_block`, rather than `.id`.